packages/relay-sdk/src/util.ts

```diff
@@ -0,0 +1,25 @@
+export function assertStatus(code: number, error: string) {
+    return (response: Response) => {
+        if (response.status !== code) {
+            throw new Error(error);
+        }
+
+        return response;
+    };
+}
+
+export function assertNotNull(value: any, error: string) {
+    if (value === null || value === undefined) {
+        throw new Error(error);
+    }
+
+    return value;
+}
+
+export function assertString(value: any, error: string) {
```

Comment:
:nit: This could be a TS assertion function instead of using `as` on the `return`
```suggestion
export function assertString(value: any, error: string) asserts value is string {
```
Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#:~:text=The%20other%20type%20of%20assertion,property%20has%20a%20different%20type.&text=Here%20asserts%20val%20is%20string,known%20to%20be%20a%20string%20.&text=assertIsString(str)%3B